src/common/metrics/interfaces.ts

```diff
@@ -7,5 +7,7 @@ export interface MetricDefinition {
   metricName: string;
   extendedMetricName: string;
   metricNameForCompare: string;
+
+  // TODO(michael): This seems to be unused. We should probably remove it!
```

Comment:
This will be used on the metric explainers, we implemented the component for each metric in advance